stripe/src/main/java/com/stripe/android/view/PaymentMethodsViewModel.kt

```diff
@@ -37,13 +37,18 @@ internal class PaymentMethodsViewModel(
             snackbarData.value = it
             snackbarData.value = null
         }
+        // auto-select newly added PaymentMethod
+        selectedPaymentMethodId = paymentMethod.id
     }
 
     internal fun onPaymentMethodRemoved(paymentMethod: PaymentMethod) {
         createSnackbarText(paymentMethod, R.string.removed)?.let {
             snackbarData.value = it
             snackbarData.value = null
         }
+        if (paymentMethod.id == selectedPaymentMethodId) {
+            selectedPaymentMethodId = null
```

Comment:
why do we need this?

Reply:
this was mainly for housekeeping, i.e. it doesn't make sense to have a stored `selectedPaymentMethodId` that doesn't exist. I can remove if you'd prefer

Reply:
Ah, nice.